### PR TITLE
Fixed typos on the README.md

### PR DESCRIPTION
**Pull request by jamgar**
Change 'Singed out.' to 'Signed out.'. Removed the extra 'add' in 'rails g migration add add_session_token_to_users...'

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -524,7 +524,7 @@ class SessionsController < ApplicationController
 
   def destroy
     logout
-    redirect_to root_path, notice: "Singed out."
+    redirect_to root_path, notice: "Signed out."
```
Comment (stevepolitodesign) [inline README.md:527]: Good catch! Are you also able to update this in [the controller](https://github.com/stevepolitodesign/rails-authentication-from-scratch/blob/c9de70f76cbae7391c18c73d0c66b0f6bdeb65c4/app/controllers/sessions_controller.rb#L14)?

File: README.md [hunk 2]
```diff
@@ -1326,7 +1326,7 @@ You can read more about session replay attacks [here](https://binarysolo.chapter
 1. Add a session_token column to users table.
 
 ```bash
-rails g migration add add_session_token_to_users session_token:string
+rails g migration add_session_token_to_users session_token:string
```
Comment (stevepolitodesign) [inline README.md:1329]: Another good catch!

**Review by stevepolitodesign — COMMENTED** [review]
@jamgar thank you for your help here! I just had one comment about fixing a typo in a controller. 

Once that's addressed I can merge this in 👍

File: README.md (hunk 1 above)
Comment (jamgar) [reply README.md:527]: Updated the controller.

**Review by stevepolitodesign — APPROVED** [review]
@jamgar thanks for the help here! I just [squashed both commits into one](https://thoughtbot.com/blog/git-interactive-rebase-squash-amend-rewriting-history) since that's best practice, but not a requirement of making a PR.

**Pull request merged**